pkg/apis/serving/v1beta1/trained_model.go

```diff
@@ -27,7 +27,7 @@ import (
 // +kubebuilder:printcolumn:name="URL",type="string",JSONPath=".status.url"
 // +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
-// +kubebuilder:resource:path=trainedmodel,shortName=tm
+// +kubebuilder:resource:path=trainedmodels,shortName=tm
```

Comment:
Reflecting on this, I kind of wish we just used the name "models". Trained or Servable is implied by nature of being part of the serving.kubeflow.org API group.